### ISSUE TITLE
Proxying fails

### ISSUE THREAD

**Issue by epicfacethe3rd** [HolyAntimony/PluralKitMC#1]
Not sure exactly what's causing this.
Every time after running  an autoproxy command, the server will throw this error in the logs
```[03:34:24] [Async Chat Thread - #4/ERROR]: Could not pass event AsyncPlayerChatEvent to PluralKitMC v0.4.5
java.lang.NullPointerException: null
	at com.omnipico.pluralkitmc.UserCache.getMemberById(UserCache.java:181) ~[?:?]
	at com.omnipico.pluralkitmc.PluralKitData.getProxiedUser(PluralKitData.java:228) ~[?:?]
	at com.omnipico.pluralkitmc.ProxyListener.onChat(ProxyListener.java:52) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor75.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.5.jar:git-Paper-762]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[patched_1.16.5.jar:git-Paper-762]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.5.jar:git-Paper-762]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:624) ~[patched_1.16.5.jar:git-Paper-762]
	at io.papermc.paper.adventure.ChatProcessor.post(ChatProcessor.java:225) ~[patched_1.16.5.jar:git-Paper-762]
	at io.papermc.paper.adventure.ChatProcessor.processingLegacyFirst(ChatProcessor.java:107) ~[patched_1.16.5.jar:git-Paper-762]
	at io.papermc.paper.adventure.ChatProcessor.process(ChatProcessor.java:65) ~[patched_1.16.5.jar:git-Paper-762]
	at net.minecraft.server.v1_16_R3.PlayerConnection.chat(PlayerConnection.java:2102) ~[patched_1.16.5.jar:git-Paper-762]
	at net.minecraft.server.v1_16_R3.PlayerConnection.c(PlayerConnection.java:2017) ~[patched_1.16.5.jar:git-Paper-762]
	at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:1955) ~[patched_1.16.5.jar:git-Paper-762]
	at net.minecraft.server.v1_16_R3.PacketPlayInChat$1.run(PacketPlayInChat.java:44) ~[patched_1.16.5.jar:git-Paper-762]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]

**Comment by epicfacethe3rd** [comment]
Plugins:
Coreprotect
Deathmessages
Essentials
EssentialsChat
EssentialsSpawn
Gsit
NametagEdit
Pistonchat
PluralkitMC
ProtocolLib
SkinsRestorer
SuperVanish
TitleManager

**Comment by Allymonies** [comment]
Thanks for the report! Could you give me your system id (the 5 digit code) so that I can test & fix this tomorrow?

**Comment by epicfacethe3rd** [comment]
Sure! `bcrip` and `gplsb` are the two that are 100% having issues and are linked correctly

**Comment by Allymonies** [comment]
Thank you, I'll look into it as soon as I can! :)

**Comment by epicfacethe3rd** [comment]
no problem!

**Comment by Allymonies** [comment]
Fixed the bug in commit 5902547, version 0.4.6. If you re-download the plugin from https://github.com/Omnipico/PluralKitMC/releases/tag/0.4.6 it should work as expected, thanks for the report!
